src/cmd.js

```diff
@@ -434,6 +434,20 @@ const none = Object.freeze({
   simulate: () => null,
 });
 
+export function flattenCmd(cmd) {
+  if (
+    cmd.type === cmdTypes.MAP ||
+    cmd.type === cmdTypes.SET_TIMEOUT ||
+    cmd.type === cmdTypes.SET_INTERVAL
+  ) {
+    return flattenCmd(cmd.nestedCmd);
+  }
+  if (cmd.type === cmdTypes.LIST) {
+    return cmd.cmds.flatMap(flattenCmd);
+  }
+  return [cmd];
+}
```

Comment:
Is there any use case to justify including `Cmd.none` values in the array? It might be more useful if the array consists only of run commands and action commands.

Reply:
I can't really think of a use case other than a testing purpose which would still be covered by checking `expect(flatCmd).toEqual([])`. I've updated to remove `Cmd.none` values 👍

Reply:
I'm torn on this. I kind of like keeping the original cmds in there. though i guess there's not much value. Why filter them out though? The general idea with a flattened list is to say it `contains` a command right? if someone wants to look for Cmd.none, we should not stop them